IM_calculation/Advanced_IM/overview_adv_IM_status.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
```

Comment:
This file (and ` check_adv_IM_status.py `) would be better in a scripts folder to differentiate between importable code and scripts.

My recommendations would be either IM_calc/scripts/adv_im/
or 
IM_calc/Adv_im/scripts/

I think I'm slightly more partial to the former.

Reply:
Sure - agreed

Reply:
Gone into IM_calculation/IM_calculation/Advanced_IM/scripts